examples/src/main/java/alluxio/cli/TestRunner.java

```diff
@@ -105,7 +104,16 @@ public static void main(String[] args) throws Exception {
    *
    * @return the number of failed tests
    */
-  private int runTests() {
+  private int runTests() throws Exception {
+    // Working directory for tests
+    mDirectory = PathUtils.concatPath(mDirectory, DEFAULT_TEST_PATH);
```

Comment:
I  prefer appending a sub directory as we delete it if it exists (safer to prevent accidental deletes)